fxbattle/fxbattle_main.cpp

```diff
@@ -97,16 +96,48 @@ int main(int argc, const char* argv[])
     });
 
     //--------------------------------------------------------------------------
+
+    auto getEnvOrDefault = [](const char *name,
```

Comment:
Can you put most of this into a statsd constructor?

Reply:
getenv - sure, but I also meant the lines 116-123

Reply:
i ended up not moving it but moved other noise into stats